index.html

```diff
@@ -1052,6 +1097,160 @@ <h4 id="Abilities">Abilities</h4>
             Other abilities can be used regardless of weapon type (see Types of Attack Roll
             section).
          </p>
+         <h4 id="Legendary Abilities">Legendary Abilities</h4>
+         <p>
+           You can unlock a legendary ability at level 1, and an additional one every 5 levels.  
```

Comment:
Do we need a legendary unlock at level 1?  Over specialisation early levels often feels bad if you go the wrong way.

Reply:
Also creating a character already feels a little overwhelming whenever we run a play test.  I think sneaking the progression in in later levels would make more sense